src/core/actor/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+file(GLOB Actor_SRC *.cpp)
+
+add_library(Actor SHARED ${Actor_SRC})
```

Comment:
Are you sure this works? libActor will refer to symbols in libEspresso (e.g. `SystemInterface::gpuBegin`), creating the same kind of dependency loop that I was seeing with libEspressoCuda when built on Mac OS X.